### DISCUSSION TITLE
Disable save on clipboard on delete (d)

### DISCUSSION THREAD

**Discussion by esrid** [LazyVim/LazyVim#4599] [Q&A] [answered]
I can't find out how to disable this option. I would like to prevent the save on clipboard when deleting

**Comment by dotfrag** [comment, marked as answer]
This is determined by the `clipboard` option here https://github.com/LazyVim/LazyVim/blob/13a4a84e3485a36e64055365665a45dc82b6bf71/lua/lazyvim/config/options.lua#L65

If you want to still be able to yank to system clipboard, you can leave that option untouched, and map `d` to `"_d`, which sends the deleted text to the special `_` "black hole" register.

**Reply by esrid** [reply]
thank you.
